composer.json

```diff
@@ -12,13 +12,15 @@
 	],
 	"minimum-stability": "RC",
 	"require": {
-		"php": "^7.3 || ^8.0",
+		"php": "^7.4 || ^8.0",
 		"codeception/lib-innerbrowser": "^1.0",
 		"codeception/codeception": "^4.0"
 	},
 	"require-dev": {
 		"codeception/module-rest": "^1.0",
-		"mezzio/mezzio": "^3.0"
+		"mezzio/mezzio": "^3.0",
+		"laminas/laminas-diactoros": "^1.8.7",
+		"container-interop/container-interop": "^1.2"
```

Comment:
I added them in `require-dev` at the time because I know that in some modules only certain assertions use some of the classes that are imported into `src/`, not necessarily all of them.
I actually added this change because as a maintainer I like that the IDE is able to recognize all classes and allows me to navigate through them without having to do `composer require` manually.
I'm also curious what the Codeception policy is in this scenario.